Resources/Prototypes/Research/experimental.yml

```diff
@@ -16,6 +16,7 @@
   - RightArmBorg
   - LeftLegBorg
   - RightLegBorg
+  - VimHarness
```

Comment:
maybe rebrand hamtr tech to "Small Animal Mechs" (including the ftl strings and stuff) or something and move this there

Reply:
critter mechs